src/main/java/com/google/step/ReadApiKey.java

```diff
@@ -1,32 +0,0 @@
-package com.google.step;
```

Comment:
what's this file and why is it deleted?

Reply:
just ignore it, i updated the branch again